README.md

```diff
@@ -43,9 +43,21 @@ out, running through [some tutorials](https://google.github.io/xls/tutorials/),
 [reporting bugs](https://github.com/google/xls/issues), and letting us know
 what you think!
 
+## Install using conda
+
+```console
```

Comment:
wonder if that's the actual right decorator

Reply:
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L6384

Reply:
if we don't have the prompt `bash` is probably better here:
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L6302